### ISSUE TITLE
Don't display error when data is expected to be delayed

### ISSUE THREAD

**Issue by FelixDQ** [tmrowco/electricitymap-contrib#2688]
If data is expected to be 24 hours delayed, then we should only display the error if the data from 24 hours ago isn't available.
![image](https://user-images.githubusercontent.com/12124251/93988030-8ceddd80-fd88-11ea-8647-8d90cc32e98c.png)
This will probably be relevant as we add more zones with delayed data. 

My thinking is that we should:

- [x]  Add an option to `zones.json` where we can tag zones as only having delayed data.
- [x]  Instead of showing an error we should link to the historical view https://github.com/tmrowco/electricitymap-contrib/pull/2393 when it has been implemented.
- [ ]  If the delayed data is actually not available then we can display an error. This will require some changes to the backend API and cache.

Step 1 & 2 should be fairly straight forward. Step 3 could be done in a separate PR.

**Comment by masquel** [comment]
Hi, let me take this one?

**Comment by FelixDQ** [comment]
Absolutely! I assigned you. Let me know if you need help with something :-)

**Comment by masquel** [comment]
@FelixDQ can you give me more information?) It's not clear enough, which option you want in `zones.json` for area and is historical view already implemented.

**Comment by FelixDQ** [comment]
> It's not clear enough, which option you want in zones.json for area

I was thinking something like this:
```javascript
{
 ...
  "delays": {
          "production": 24, // 24 hours delay
           "price": 8             // 8 hour delay           
  }
}
```

> is historical view already implemented
No, not yet. For now, I would change it to display a message saying that the data is delayed instead. Then we can add the link to the historical view later on.

**Comment by FelixDQ** [comment]
Also, the message should appear if `production` data is delayed. For now, we don't care about delays in other data types

**Comment by FelixDQ** [comment]
resolved in #2692
